rtcpeerconnection.js

```diff
@@ -147,6 +147,7 @@ function PeerConnection(config, constraints) {
     this.config = {
         debug: false,
         ice: {},
+        remoteIce: {},
```

Comment:
nit: I don't think this should be part of config. It is not configurable after all.